src/stories/layouts/Grid/Grid.js

```diff
@@ -0,0 +1,39 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component for grid layout element.
+ *
+ * @param {node} children of the component
+ * @param {string} numColumns number of columns in layout.
+ * @return {object} (
+ *   <Grid numColumns={numColumns}>
+ *      {children}
+ *    </Grid>
+ * )
+ */
+
+export const Grid = ({ children, numColumns }) => {
+    return (
+        <div className={['grid', `${numColumns}`].join(' ').trim()}>
```

Comment:
I like the idea of using the number of columns; I would just update this to `grid--${numColumns}-col` and change the `numColumns` param to a number, which would return the following modifier class: `grid--3-col`.  That way we don't need to spell out numbers which is more likely to lead to typos and then bugs. Example of update:
```
const modifierClass = numColumns ? `grid--${numColumns}-col` : '';
    return (
        <div className={['grid', modifierClass].join(' ').trim()}>
            {children}
        </div>
    );
```